tools/verify-golint.sh

```diff
@@ -35,9 +35,7 @@ PATH="${GOBIN}:${PATH}"
 
 # Install golint
 echo 'installing golint'
-pushd "${KUBE_ROOT}/tools" >/dev/null
-  GO111MODULE=on go install -mod=readonly golang.org/x/lint/golint
-popd >/dev/null
+GO111MODULE=on go install golang.org/x/lint/golint
```

Comment:
I was going to remove `GO111MODULE=on` but we don't have a go version check guarding this code, so it is safer to leave it in until we add that check.